functests/utils/consts.go

```diff
@@ -24,7 +30,25 @@ func init() {
 		PerformanceProfileName = "performance"
 	}
 
+<<<<<<< HEAD
 	NodesSelector = os.Getenv("NODES_SELECTOR")
+=======
+	if discovery.Enabled() {
+		performanceProfile, err := profiles.GetByNodeLabels(
+			map[string]string{
+				fmt.Sprintf("%s/%s", LabelRole, RoleWorkerCNF): "",
```

Comment:
My PR addresses this ad-hoc construct appearing in a lot of places, I'll be sure to add also this occurrence.